GoToProject/src/containers/DetailsScreen/index.js

```diff
@@ -21,6 +20,10 @@ const placeImg =
 const Details =
   'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
 
+const LATITUDE = 19.129579469349792;
+const LONGITUDE = -103.88428965974752;
+const placeName = 'Piedra Acampanada';
+
```

Comment:
You can call these constants from an external file.

Reply:
created a file for constants